cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -958,7 +958,7 @@ private Future<MapAndStatus<ConfigMap, KafkaRebalanceStatus>> onRebalancing(Reco
             apiClient.getUserTaskStatus(host, cruiseControlPort, sessionId)
                 .onSuccess(cruiseControlResponse -> {
                     if (cruiseControlResponse.getJson().isEmpty()) {
-                        LOGGER.infoCr(reconciliation, "Cruise Control restarted, going to generate a new proposal");
+                        LOGGER.warnCr(reconciliation, "Json data for response is empty since user task was not found, going to generate a new proposal");
```

Comment:
I would simply log:

```sh
LOGGER.warnCr(reconciliation, "User task {} not found, going to generate a new proposal", sessionId);
```

Also, so we want to add a comment on why this could happen?

Reply:
Done. I have added the commment which you are using for Topic operator